kolibri/plugins/learn/assets/src/vue/content-render.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+
+  <div>
+    <h3>
+      {{ title }}
+    </h3>
+    <div v-el:container>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  const Kolibri = require('kolibri');
+
+  module.exports = {
+    props: [
+      'channelId',
+      'contentId',
+      'kind',
+      'extension',
+      'title',
+    ],
+    computed: {
+      contentType() {
+        return `${this.kind}/${this.extension}`;
+      },
+    },
+    created() {
+      this.contentData = { url: '#' };
```

Comment:
okay. note that you can specify default prop values: https://vuejs.org/guide/components.html#Prop-Validation